hi-ml-histopathology/src/histopathology/preprocessing/create_panda_tiles_dataset.py

```diff
@@ -217,14 +246,69 @@ def main(panda_dir: Union[str, Path], root_output_dir: Union[str, Path], level:
 
     logging.info("Merging slide files in a single file")
     merge_dataset_csv_files(output_dir)
+    logfile.close()
 
 
 if __name__ == '__main__':
-    main(panda_dir="/tmp/datasets/PANDA",
-         root_output_dir="/datadrive",
-         level=1,
-         tile_size=224,
-         margin=64,
-         occupancy_threshold=0.05,
-         parallel=True,
-         overwrite=False)
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--container-dir",
```

Comment:
What do you mean by 'container' in this context? Why not pass the PANDA folder directly?

Reply:
I meant "folder on which an Azure blob container is mounted". I can't remember why I switched to specifying the container, but I'll revert back.